### PR TITLE
DNS - Update with management code sample

### PR DESCRIPTION
**Pull request by simonaco**

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/dns.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-8/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C15%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707151725560611-92%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/dns.md [hunk 1]
```diff
@@ -13,24 +13,36 @@ ms.devlang: nodejs
...
-
-
-### Management
 ```bash
 npm install azure-arm-dns
 ```
 
+### Example
+
+Authenticate, create client and list zones.
```
Comment (johnpapa) [inline docs-ref-services/dns.md]: > Authenticate, create the client and list the zones.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/dns.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-8/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707171638329958-92%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/dns.md [hunk 2]
```diff
@@ -13,24 +13,36 @@ ms.devlang: nodejs
...
-# Azure DNS modules for Node.js
+# Azure DNS packages for Node.js
 
 ## Overview
 
-## Install the modules with npm
+## Management package
 
-Use npm to install the Azure DNS modules for Node.js
+### Install npm module
```
Comment (johnpapa) [inline docs-ref-services/dns.md]: Missing the 

> Install the Azure DNS npm module

File: docs-ref-services/dns.md [hunk 3]
```diff
@@ -13,24 +13,36 @@ ms.devlang: nodejs
...
 
-##Example
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const DNSManagement = require('azure-arm-dns');
+
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const client = new DNSManagement(credentials, 'your-subscription-id');
```
Comment (johnpapa) [inline docs-ref-services/dns.md]: move subscription to a variable.

ideally like this ...


```javascript
const msRestAzure = require('ms-rest-azure');
const authorizationManagement = require('azure-arm-authorization');
  		  
subscriptionId = 'your-subscription-id';

// more code .....

```

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/dns.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-8/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180058016899-92%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/dns.md [hunk 4]
```diff
@@ -13,24 +13,40 @@ ms.devlang: nodejs
 ms.service: DNS
 ---
 
-# Azure DNS modules for Node.js
+# Azure DNS Modules for Node.js
 
 ## Overview
```
Comment (johnpapa) [inline docs-ref-services/dns.md]: Needs an overview @simonaco

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/dns.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-8/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180409397623-92%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/dns.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-8/docs-ref-services/dns.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180422018001-92%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM

#sign-off

**Pull request merged**